### PR TITLE
Update binder link to point to the current stable version

### PR DESCRIPTION
**Pull request by CAM-Gerlach**
# Pull Request

## Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder-docs/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 4.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [x] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

Updates the Binder link various places to point to the current, fixed link.

### Issue(s) Resolved

Part of spyder-ide/website-spyder#210

### CODE REVIEW

File: doc/installation.rst [hunk 1]
```diff
@@ -24,10 +24,10 @@ Try Spyder online
...
 Want to try out Spyder without installing it?
 With `Binder`_ you can work with a fully functional copy of Spyder online that runs right in your web browser, no installation needed.
-Just visit the `Spyder page on Binder`_ to get started using Spyder now.
+Visit the `Spyder Binder`_ to get started using Spyder.
 
 .. _Binder: https://mybinder.org/
-.. _Spyder page on Binder: https://mybinder.org/v2/gh/spyder-ide/spyder/5.x?urlpath=/desktop
+.. _Spyder Binder: https://mybinder.org/v2/gh/spyder-ide/binder-environments/spyder-stable?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fspyder-ide%252FSpyder-Workshop%26urlpath%3Ddesktop%252F%26branch%3Dmaster
 
 .. image:: /images/installation/installation-spyder-binder.png
```
Comment (ccordoba12) [inline doc/installation.rst:32]: Perhaps we should update this image to reflect how Spyder 5 looks in Binder now? Not a blocker for this PR though, so I'll leave the decision to you.

**Review by ccordoba12 — COMMENTED** [review]
Looks good to me, thanks @CAM-Gerlach. I only left an enhancement suggestion for you below.

File: doc/installation.rst (hunk 1 above)
Comment (CAM-Gerlach) [reply doc/installation.rst:32]: Sure, I went ahead and added an updated screenshot

Comment (ccordoba12) [reply doc/installation.rst:32]: Thanks!

**Review by ccordoba12 — APPROVED** [review]

**Review by jitseniesen — APPROVED** [review]

**Pull request merged**